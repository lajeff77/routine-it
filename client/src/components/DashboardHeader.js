import React from "react";
import { Box, Typography } from "@mui/material";


const DashboardHeader =  () => {
    return (
        <div>
            <Box>
                <Box>
                    <Typography variant="h1" sx={{textAlign: 'center', margin: '10px'}}>My Dashboard</Typography>
                </Box>
            </Box>
        </div>
    )
}

export default DashboardHeader;