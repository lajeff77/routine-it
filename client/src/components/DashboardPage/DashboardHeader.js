import React from "react";
import { Box, Typography } from "@mui/material";


const DashboardHeader =  () => {
    return (
        <div>
            <Box>
                <Box>
                    <Typography variant="h1" sx={{textAlign: 'center', margin: '30px'}}>My Dashboard</Typography>
                </Box>
            </Box>
        </div>
    )
}

export default DashboardHeader;