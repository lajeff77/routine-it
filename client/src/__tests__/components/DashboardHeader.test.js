import { render } from "@testing-library/react";
import {expect, test} from '@jest/globals';
import React from "react";
import DashboardHeader from "../../components/DashboardPage/DashboardHeader";


describe(DashboardHeader.name, ()=>{
    test('DashboardHeader matches snapshot', () => {
        const {container} = render(<DashboardHeader/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});