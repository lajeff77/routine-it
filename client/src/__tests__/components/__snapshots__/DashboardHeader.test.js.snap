// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHeader DashboardHeader matches snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiBox-root css-0"
    >
      <h1
        class="MuiTypography-root MuiTypography-h1 css-3ns5jg-MuiTypography-root"
      >
        My Dashboard
      </h1>
    </div>
  </div>
</div>
`;
